content/en/blog/_posts/2021-04-08-introducing-indexed-jobs.md

```diff
@@ -0,0 +1,89 @@
+---
+title: "Introducing Indexed Jobs"
+date: 2021-05-05
+slug: introducing-indexed-jobs
+---
+
+**Author:** Aldo Culquicondor (Google)
+
+Once you have containerized a non-parallel {{< glossary_tooltip text="Job" term_id="job" >}},
+it is quite easy to get it up and running on Kubernetes without modifications to
+the binary. But when running parallel distributed Jobs, in most cases, you had
```

Comment:
In most cases, when running parallel distributed Jobs, you had to ...

Reply:
Done